primary/src/grpc_server/mod.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+use std::time::Duration;
+
+use self::validator::NarwhalValidator;
+use crate::BlockCommand;
+use tokio::sync::mpsc::Sender;
+use tonic::transport::Server;
+use tracing::error;
+use types::ValidatorServer;
+
+mod validator;
+
+pub struct GrpcServer {
```

Comment:
+1

I would probably go with something neutral (e.x `ConsensusAPIGrpc` , `ConsensusGrpcServer`) and avoid using the `validator` etc, since we already have services named like this (e.x the Validator API)